packages/jest-cli/src/plugins/quit.js

```diff
@@ -9,15 +9,7 @@
 import {BaseWatchPlugin} from 'jest-watcher';
 
 class QuitPlugin extends BaseWatchPlugin {
-  isInternal: true;
-
-  constructor(options: {
-    stdin: stream$Readable | tty$ReadStream,
-    stdout: stream$Writable | tty$WriteStream,
-  }) {
-    super(options);
-    this.isInternal = true;
-  }
+  isInternal: true = true;
```

Comment:
Should be boolean instead of true, here and in other places

Reply:
In here it's fine, I'd say. It can never be `false`, thus `true` is just stricter. The parent class (`BaseWatchPlugin`) needs to be `boolean`, though